### PR TITLE
Add support for retrieve posting list

### PR DESCRIPTION
**Pull request by douglasfarinelli**
This pull requests implements the 'solicitaXmlPlp' method. [ref. page 27](http://www.corporativo.correios.com.br/encomendas/sigepweb/doc/Manual_de_Implementacao_do_Web_Service_SIGEP_WEB.pdf)

This method returns all information after the post has been successfully completed.

### CODE REVIEW

**Comment by lamenezes** [comment]
@douglasfarinelli  thanks! We'll check your PR as soon as we can :tada:

**Comment by douglasfarinelli** [comment]
It's broken because needs updates wsdl. :disappointed:

**Comment by douglasfarinelli** [comment]
Fixed :point_up_2:

**Comment by douglasfarinelli** [comment]
[important] Python 3.5.0 is not supported to run mypy and so the travis-ci was failing. Fix in c54dd1e. Maybe it makes sense, but I think it's worth someone's remark.

ref: https://github.com/python/mypy/issues/4714

File: correios/models/posting.py [hunk 1]
```diff
@@ -497,6 +498,52 @@ def _validate_weight(cls, weight, service: Optional[Union[Service, str, int]] =
             raise exceptions.InvalidMaxPackageWeightError(message)
 
 
+class Receipt:
+
```
Comment (osantana) [inline correios/models/posting.py]: no blank line here...

File: correios/models/posting.py [hunk 2]
```diff
@@ -718,6 +770,38 @@ def closed(self):
         return self.number is not None
 
 
+PostalUnit = collections.namedtuple('PostalUnit', 'code description')
```
Comment (osantana) [inline correios/models/posting.py]: Please, create a "dumb" real class here...

```python
class PostalUnit:
   def __init__(self, code:str, description: str) -> None:
       self.code = code
       self.description = description
```

File: correios/models/posting.py [hunk 3]
```diff
@@ -718,6 +770,38 @@ def closed(self):
...
+
+
+class PostInfo:
+
```
Comment (osantana) [inline correios/models/posting.py]: no blank line here...

File: correios/models/posting.py [hunk 4]
```diff
@@ -718,6 +770,38 @@ def closed(self):
...
+        self,
+        postal_unit: PostalUnit,
+        posting_list: PostingList,
+        value: Union[Decimal, float, str]
+    ) -> None:
+        self.postal_unit = postal_unit
+
+        self.posting_list = posting_list
+
+        self.real_value = value
```
Comment (osantana) [inline correios/models/posting.py]: too many blank lines here...

**Comment by jpbonson** [comment]
This PR [1] will fix the problem you had at CI :slightly_smiling_face: 

[1] https://github.com/olist/correios/pull/128

**Pull request merged**